src/main/java/thewaypointers/trafficsimulator/JunctionLocationTestStarter.java

```diff
@@ -18,9 +13,10 @@
 
     public static void main(String[] args){
 
-        IStateProvider simulation = new JunctionTestProvider();
-        mainFrame=new MainFrame(simulation.getNextState(0));
-        TrafficSimulatorManager.setMainFrame(mainFrame);
-        TrafficSimulatorManager.setSimulation(simulation);
+        TrafficSimulatorManager trafficSimulatorManager = new TrafficSimulatorManager();
+        IStateProvider simulation = new RoadNetworkProvider();
```

Comment:
Because the JuntionTestProvider will cause problems in feat-48, we get a list of null for junction, so we can't click any junction. But it works well on RoadNetworkProvider. Mengzhu and I just find this problem and use this as a test of feat-48